datus/storage/document/nav_resolver/mkdocs_resolver.py

```
@@ -0,0 +1,112 @@
+logger = get_logger(__name__)
+
+
+class MkDocsResolver(BaseNavResolver):
+    """Resolves nav_path from MkDocs mkdocs.yml configuration."""
+
+    def resolve(
+        self,
+        config_content: str,
+        file_paths: List[str],
+        content_root: str,
+        extra_context: Optional[Dict[str, Any]] = None,
```

Ruff (0.14.14)

[warning] Unused method argument: `extra_context`

(ARG002)